api/api.js

```diff
@@ -64,6 +64,11 @@ function init (app) {
       }
 
       case 'createTournament': {
+        try {
+          require('../src/account_util').require_logged_in_user(req, res)
```

Comment:
This function is not intended for use in the api, redirecting only makes sense for get request. The previous implementation (`if (user === null)...`) was correct